go.mod

```diff
@@ -46,6 +47,7 @@ require (
 	github.com/multiformats/go-varint v0.0.6
 	github.com/prometheus/common v0.30.0 // indirect
 	github.com/prometheus/procfs v0.7.3 // indirect
+	github.com/raulk/go-watchdog v1.2.0
```

Comment:
that a problem?

Reply:
It's hidden behind the cgo flag (https://github.com/libp2p/go-libp2p-connmgr/pull/103).